src/ArrayHelper.php

```diff
@@ -1330,6 +1330,29 @@ public static function getObjectVars(object $object): array
         return get_object_vars($object);
     }
 
+    /**
+     * Replace key in array.
+     *
+     * @param array $array Source array.
+     * @param int|string $from Key to replace.
+     * @param int|string $to Key to replace with.
+     *
+     * @return array The result array.
+     */
+    public static function replaceKey(array $array, int|string $from, int|string $to): array
```

Comment:
How about nested array keys provided with `.` syntax?

Reply:
I suggest we leave this for future improvements.

Reply:
OK/